src/Hyperion.Tests/UnsafeDeserializationExclusionTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.IO;
+using Hyperion.Extensions;
+using Xunit;
+
+namespace Hyperion.Tests
+{
+    public class UnsafeDeserializationExclusionTests
+    {
+        [Fact]
+        public void CantDeserializeANaughtyType()
+        {
+            //System.Diagnostics.Process p = new Process();
+            var serializer = new Hyperion.Serializer();
+            var di =new System.IO.DirectoryInfo(@"c:\");
```

Comment:
Not sure if this unit test will play well in other OS runs... May need to make this something more platform friendly.